data/aadii0408.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Aaditya Champaneri || Aadi ",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  
+  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities"],
+  
+  "avatar": "https://github.com/aadii0408.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aadii0408",
```

Comment:
It seems like you've added a wrong URL as this account doesn't exists. Please update it with the correct one! 🛠

![image](https://user-images.githubusercontent.com/88786642/215285131-a8a864ad-0453-49ad-bb14-921ce5247745.png)

Reply:
It's not a PR blocker. Then can raise another PR to fix that. 

Thank you